pkg/logstructured/sqllog/sql.go

```diff
@@ -468,8 +474,6 @@ func (s *SQLLog) startWatch() (chan interface{}, error) {
 }
 
 func (s *SQLLog) poll(result chan interface{}, pollStart int64) {
-	s.currentRev = pollStart
```

Comment:
I would probably create a new local currentRev var, instead of just changing pollStart to be used for this purpose. Continuing to compare against and update pollStart is confusing, since it is not being used as the 'start' any longer.